src/platform_impl/ios/mod.rs

```diff
@@ -109,8 +110,6 @@ pub enum OsError {}
 
 impl fmt::Display for OsError {
     fn fmt(&self, _: &mut fmt::Formatter<'_>) -> fmt::Result {
-        match self {
-            _ => unreachable!(),
-        }
+        unreachable!()
```

Comment:
`unreachable!()` typically means something the compiler hasn't been able to deduce... This type is `pub`, are we sure no-one is ever able to call `Display` on `OsError`?

Reply:
Yeah, I'll add some stub.